### ISSUE TITLE
Strange behavior of "align_comma_semicolon" with alignment of struct declaration

### ISSUE THREAD

**Issue by anton-dos** [TheClams/SystemVerilog#20]
With "sv.align_comma_semicolon" : true
```
typedef struct packed {
    logic a  ;
    logic aaa;
} struct_t;
```
With "sv.align_comma_semicolon" : false
```
typedef struct packed {
    logic a;··
    logic aaa;
} struct_t;
```
Note: · represents a space.

**Comment by DeeeeLAN** [comment]
What are you expecting it to be?

**Comment by anton-dos** [comment]
When aligning module declaration, there are no spaces at the end of lines

**Comment by TheClams** [comment]
Fixed in v3.0.5 (ST4)
